src/utils/requests/loadRainGeoJson.ts

```diff
@@ -2,8 +2,5 @@ import { ExtendedFeatureCollection } from 'd3-geo';
 import { requests } from '../requestUtil';
 
 export const loadRainGeoJson = async (): Promise<ExtendedFeatureCollection> => {
-  const dataUrl =
-    process.env.NEXT_PUBLIC_RAIN_DATA_URL || 'https://tsb-trees.s3.eu-central-1.amazonaws.com/weather_light.geojson.gz';
-
-  return await requests<ExtendedFeatureCollection>(dataUrl);
+  return await requests<ExtendedFeatureCollection>(process.env.NEXT_PUBLIC_RAIN_DATA_URL!);
```

Comment:
I know Next.js throws during build when we try to access NEXT_PUBLIC_WHATEVER but can we still have a check for it? So we don't need to do the nasty `VAR!` non null assertion.

```js
if(!process.env.NEXT_PUBLIC_RAIN_DATA_URL) throw new Error("env var NEXT_PUBLIC_RAIN_DATA_URL should be defiend in your environment")
```

Reply:
Sure thing